example/android/app/build.gradle

```diff
@@ -11,9 +11,9 @@ react {
     //   The root of your project, i.e. where "package.json" lives. Default is '../..'
     // root = file("../../")
     //   The folder where the react-native NPM package is. Default is ../../node_modules/react-native
-    // reactNativeDir = file("../../node_modules/react-native")
+    reactNativeDir = file("../../../node_modules/react-native")
```

Comment:
Explanation in https://github.com/Expensify/react-native-live-markdown/pull/529/files#r1819206798